tdrs-backend/tdpservice/users/api/authorization_check.py

```diff
@@ -26,11 +27,14 @@ def get(self, request, *args, **kwargs):
             auth_params = {
                 "authenticated": True,
                 "user": serializer.data,
+                "csrf": csrf.get_token(request),
```

Comment:
Are we sure this is secure?

Reply:
Should be, this value was already to be exposed to the client in a cookie. This wouldn't change the overall availability of this value from what it was before, though I'll be looking into it.